.github/workflows/test.sh

```diff
@@ -5,3 +5,32 @@ set -e -u -o pipefail
 VERSION=${1:-stable}
 
 rustup run ${VERSION} cargo test --verbose
+
+# CLI tests
```

Comment:
I think for maintenance it would be nice to have this section in its own cli_tests.sh file, that is called by test.sh (so the .yml remains unchanged, but this work is in its own place).